Dockerfile

```
@@ -1,10 +1,10 @@
-FROM crystallang/crystal:1.10.1-alpine as builder
+FROM crystallang/crystal:1.12-alpine as builder
 WORKDIR /tmp/build
 COPY . /tmp/build
 RUN shards build --production
```

Might be worth to enable multi-threading (MT).

```suggestion
RUN shards build --production -Dpreview_mt
```